test/suite/test_app_thread_evict01.py

```diff
@@ -81,13 +84,22 @@ def test_app_thread_evict01(self):
         cursor.set_value(str(key))
         cursor.insert()
 
-        for i in range(500):
-            self.insert_range(self.uri, i)
 
-        cursor.set_key(350)
-        self.assertEquals(cursor.search(), 0)
+        while retries < retry_limit:
```

Comment:
This will let us drop the `retry` and `retry_limit` variables
```suggestion
        # For our target stat to be incremented we need our application thread to evict a page, but the 
        # this is probabilistic as the application thread is always racing against the internal eviction threads.
        # Give the application thread a few chances to beat the internal thread 
        for _ in range(0, 10):
```

Reply:
Thanks for the suggestions, Fixed